store/src/test/java/org/apache/rocketmq/store/ConsumeQueueTest.java

```diff
@@ -250,7 +250,11 @@ public void testPutMessagePositionInfo_buildCQRepeatedly() throws Exception {
             for (int i = 0; i < totalMessages; i++) {
                 putMsg(messageStore);
             }
-            Thread.sleep(5);
+
+            // Wait consume queue build finish.
+            while (messageStore.dispatchBehindBytes() > 0) {
+                Thread.sleep(5);
+            }
```

Comment:
Waiting for the desired condition with a max time may be better, like `awaitility`.